src/test/ui/lint/command-line-lint-group-forbid.stderr

```diff
@@ -0,0 +1,10 @@
+error: variable `_InappropriateCamelCasing` should have a snake case name such as `_inappropriate_camel_casing`
+  --> $DIR/command-line-lint-group-forbid.rs:14:9
+   |
+14 |     let _InappropriateCamelCasing = true;
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: `-F bad-style` implies `-F non-snake-case`
```

Comment:
I keep being surprised by the ordering here. What do you think about saying "`-F non-snake-case` implied by `-F bad-style` or something?